truffle-config.js

```diff
@@ -68,6 +47,51 @@ const manualNetworks = {
   },
 }
 
+/**
+ * You shouldn't change anything below this line to preserve this Truffle Box's automatic handling of target realms,
+ * chains, networks, compiler versions, etc.
+ *
+ * If you want to configure anything related to networks, please customize the `manualNetworks` object above.
+ *
+ * Target realms (ethereum / boba / celo / conflux / etc.) are read from multiple possible sources, in this order:
+ *  1. `--network <realm>.<chain>` argument
+ *  2. `WITNET_EVM_REALM` environment variable
+ *  3. default (ethereum)
+ */
+const witnetSettings = require("./node_modules/witnet-ethereum-bridge/migrations/witnet.settings")
+let realm = process.env.WITNET_EVM_REALM ? process.env.WITNET_EVM_REALM.toLowerCase() : "default"
+const args = process.argv.join("=").split("=")
+const networkIndex = args.indexOf("--network")
+if (networkIndex >= 0) {
+  realm = (args[networkIndex + 1] || "default").split(".")[0]
+}
+console.info(`Configuring "${realm}" target realm...
+==============================${"=".repeat(realm.length)}
+> Default realm is "ethereum"
+> Target realms and chains can be specified with "--network <realm>.<chain>"
+> Available realms and chains can be found in "migrations/2_user_contracts.js"`)
```

Comment:
Shouldn't it refer to either `'migrations/1_witnet_core.js' or './node_modules/witnet-solidity-bridge/migraitons/witnet.settings' ?

Reply:
` WITNET_EVM_REALM`  will only be taken into account if no ` network`  is specified by argument. Therefore, it would only apply to "development"-named networks within each realm section in the "witnet.settings" file. In case this was not the intended logic, another option would be to extract realm from the network (or default it to "default", if necesarry) only if the ` WITNET_EVM_REALM` is not defined.

Reply:
True, nice catch!